.goreleaser.yml

```diff
@@ -15,6 +15,9 @@ builds:
     goarch:
       - amd64
     ldflags:
+      # the following line will inject the current git commit and git tag in the binary during the build process.
+      # .ShortCommit and .Version are template variables that will be set during the GoReleaser run
+      # The "-X" go flag injects the strings into the two global variables GitCommit and Version
       - -s -w -X github.com/openshift/osdctl/cmd.GitCommit={{.ShortCommit}} -X github.com/openshift/osdctl/cmd.Version={{.Version}}
```

Comment:
this is great, but a crosslink  to this would make it better: the lines I want to have the crosslinks in
https://github.com/openshift/osdctl/blob/1447240f5a5f8b38288ee6b41a971d34e762d34d/cmd/version.go#L20-L24

if we are already here, maybe split the ldflags to
```
- -s
- -w
- -X ...
- -X ...
```
this way you can comment on each of them for the change

Reply:
the link to the -X is https://pkg.go.dev/cmd/link

Reply:
done